mkosi/initrd/__main__.py

```diff
@@ -0,0 +1,123 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import argparse
+import os
+import platform
+import shutil
+import tempfile
+from pathlib import Path
+
+from mkosi.config import __version__
+from mkosi.log import log_setup
+from mkosi.run import find_binary, run, uncaught_exception_handler
+from mkosi.types import PathString
+
+
+@uncaught_exception_handler()
+def main() -> None:
+    log_setup()
+
+    parser = argparse.ArgumentParser(
+        prog="mkosi-initrd",
+        description="Build initrds or Unified Kernel Images using mkosi",
+        allow_abbrev=False,
+        usage="mkosi-initrd [options...]",
+    )
+
+    parser.add_argument(
+        "--kernel-version",
+        metavar="KERNEL_VERSION",
+        help="Kernel version string",
+        default=platform.uname().release
+    )
+    parser.add_argument(
+        "-t", "--format",
+        choices=["cpio", "uki", "directory"],
```

Comment:
Can you use `OutputFormat.cpio` and such here instead of the raw strings? That makes everything a bit more type safe.